files/bootstrap.sh

```diff
@@ -35,6 +35,7 @@ function print_help {
   echo "--local-disks Setup instance storage NVMe disks in raid0 or mount the individual disks for use by pods [mount | raid0]"
   echo "--mount-bfs-fs Mount a bpffs at /sys/fs/bpf (default: true, for Kubernetes 1.27+; false otherwise)"
   echo "--pause-container-account The AWS account (number) to pull the pause container from"
+  echo "--pause-container-repository-name The AWS ECR repository name to pull the pause container from"
```

Comment:
nit: this doesn't need to be an `AWS ECR repository`, it can be any image registry.